security/v1/types.go

```diff
@@ -67,6 +67,14 @@ type SecurityContextConstraints struct {
 	AllowHostPID bool `json:"allowHostPID" protobuf:"varint,11,opt,name=allowHostPID"`
 	// AllowHostIPC determines if the policy allows host ipc in the containers.
 	AllowHostIPC bool `json:"allowHostIPC" protobuf:"varint,12,opt,name=allowHostIPC"`
+	// DefaultAllowPrivilegeEscalation controls the default setting for whether a
+	// process can gain more privileges than its parent process.
+	// +optional
+	DefaultAllowPrivilegeEscalation *bool `json:"defaultAllowPrivilegeEscalation,omitempty" protobuf:"varint,22,rep,name=defaultAllowPrivilegeEscalation"`
```

Comment:
> given where the upstream object ended up, I'm not sure we want two fields for this...

I'm in favor of being close to upstream as much as possible.

> would we ever set defaultAllowPrivilegeEscalation=false without also setting allowPrivilegeEscalation=false or vice versa?

No, we can't set `defaultAllowPrivilegeEscalation=false` and `allowPrivilegeEscalation=false` as it will be rejected by validation code.

But, yes, it's possible to have `defaultAllowPrivilegeEscalation=false` and `allowPrivilegeEscalation=true` to define strict defaults but allow to explicitly override them.

> which I think made defaultAllowPrivilegeEscalation much less useful

Maybe it became less useful, but it still can be used in the case described above. Also, explicit separation between rule and defaulter is a right move.

Reply:
I second @php-coder here

Reply:
ok